### PR TITLE
Pick translation

### PR DESCRIPTION
**Pull request by Adnanog**

### CODE REVIEW

File: client/src/components/Dashboard.js [hunk 1]
```diff
@@ -13,10 +13,51 @@ export class Dashboard extends Component {
...
+      body: JSON.stringify({
+        document_id: id
+      }),
+      headers: {
+        Authorization: `Bearer ${sessionStorage.getItem("token")}`,
+        "Content-Type": "application/json"
+      }
+    };
+
+    fetch("http://localhost:4000/api/translations", userRequest)
```
Comment (llh1) [inline client/src/components/Dashboard.js]: This will not work in Heroku, this url is okay just locally. Remove the localhost bit.

**Review by llh1 — CHANGES_REQUESTED** [review]

File: client/src/components/Dashboard.js (hunk 1 above)
Comment (llh1) [inline client/src/components/Dashboard.js]: Moreover, you could extract all this function in client/src/api/translations.js file

**Review by llh1 — CHANGES_REQUESTED** [review]

File: client/src/components/Dashboard.js (hunk 1 above)
Comment (Adnanog) [reply client/src/components/Dashboard.js]: it is done

Comment (Adnanog) [reply client/src/components/Dashboard.js]: I know it is convenient to do that but when I am trying to do so, the pickDocument  contains 2 functions which are needed in the original dashboard component (handleErrors and setDocument) which cause many problems when click on Pick Document button.

**Review by llh1 — APPROVED** [review]

**Pull request merged**